test/fixtures/config/patients.js

```diff
@@ -0,0 +1,14 @@
+const faker = require('faker');
+
+module.exports = {
+  generate: () => {
+    return {
+      'id' : faker.random.uuid(),
+      'first_name' : faker.name.firstName(),
+      'last_name' : faker.name.lastName(),
+      'birth_date' : faker.date.past(),
+      'sex' : faker.random.arrayElement(['m', 'f']),
+      'mrn' : faker.random.number(),
```

Comment:
is this actually a number?  I think at least it's a string that might be a number so `${ faker.random.number() }`?

Reply:
yup, looks like it gets converted to a string by the backend 👍